platform/stacks/portal.stack.yml

```diff
@@ -9,17 +9,20 @@ networks:
 services:
 
   portal:
-    image: appcelerator/amp-ui:1.0.0
+    image: appcelerator/portal:${TAG:-0.10.1}
     networks:
       - default
     deploy:
       mode: replicated
       replicas: 1
       labels:
         io.amp.role: "infrastructure"
+      placement:
+        constraints:
+          - node.role == ${NODE_ROLE:-worker}
     environment:
       SERVICE_PORTS: "3333"
-      VIRTUAL_HOST: "http://ui.*,https://ui.*,http://cloud.*,http://local.*,https://cloud.*,https://local.*"
+      VIRTUAL_HOST: "http://portal.*, https://portal.*,http://ui.*,https://ui.*,http://cloud.*,http://local.*,https://cloud.*,https://local.*"
```

Comment:
Can you remove the references to `portal` and `ui` ... it's our portal, but we want to access it at `cloud.*`. Thanks.